pkg/validation/cluster.go

```diff
@@ -135,11 +135,6 @@ func ValidateClusterUpdate(ctx context.Context, newCluster, oldCluster *kubermat
 		allErrs = append(allErrs, errs...)
 	}
 
-	portRangeFld := field.NewPath("componentsOverride", "apiserver", "nodePortRange")
-	if err := ValidateNodePortRange(newCluster.Spec.ComponentsOverride.Apiserver.NodePortRange, portRangeFld); err != nil {
```

Comment:
what happened with ValidateNodePortRange?

Reply:
It moved and was a duplication. Its now at: 
https://github.com/kubermatic/kubermatic/pull/8346/files#diff-37f3a956a5e4b5bdf832aafa72b7897ef5697e5e1ded312682119b1452e8158aR104